dirsrvtests/tests/suites/schema/schema_reload_test.py

```diff
@@ -56,18 +54,16 @@ def test_schema_reload_with_searches(topo):
 
     # Run a schema_reload tasks while searching for our user.Since
     # this is a race condition, run it several times.
-
-    topo.standalone.tasks.schemaReload(schemadir=topo.standalone.schemadir,
-                                              args={TASK_WAIT: False})
+    schema = Schema(topo.standalone)
+    task = schema.reload(schema_dir=topo.standalone.schemadir)
 
     # While we wait for the task to complete search for our user
     search_count = 0
     while search_count < 10:
     # Now check the user is still being returned
     # Check if task is complete
         assert user.exists()
-        task_entry = topo.standalone.tasks.entry
-        if topo.standalone.tasks.checkTask(task_entry)[0]:
+        if task.get_exit_code() == 0:
```

Comment:
didn't use task.wait() here as the test needs it to be a race condition. Also made changes in other tests.